sdk/core/core-amqp/src/ConnectionContextBase.ts

```diff
@@ -137,7 +137,9 @@ export module ConnectionContextBase {
     const userAgent = parameters.connectionProperties.userAgent;
     if (userAgent.length > Constants.maxUserAgentLength) {
       throw new Error(
-        `The user-agent string cannot be more than ${Constants.maxUserAgentLength} characters in length.` +
+        `The user-agent string cannot be more than ${
+          Constants.maxUserAgentLength
+        } characters in length.` +
```

Comment:
Can you check if you are using Prettier with the same settings as what is defined in the repo? I recall that this particular line keeps changing based on who is touching this file :) Please check with @ShivangiReja

Reply:
It looks like @ShivangiReja is using the `.prettierrc.json` file defined in eventhubs. I seem to be having one on my local from maybe the main repo.
Is there a link to a standard settings file to use?
@KarishmaGhiya